src/ApplicationCore/Entities/FraudProtectionApiModels/AccountProtection/Response/ResponseSuccess.cs

```diff
@@ -13,5 +13,7 @@ public class ResponseSuccess : Response
         public string TransactionReferenceId { get; set; }
 
         public Enrichments<AccountProtectionDeviceAttributes> Enrichments { get; set; }
+
+        public IDictionary<string, IList<object>> Diagnostics { get; set; }
```

Comment:
Should we use IEnumerable instead of IList, to be more generic, in both files?

Reply:
Sure, updated.